Sources/Rendering/OpenGL/Texture/test/testCreateCubeFromRawTexture.js

```diff
@@ -121,12 +103,12 @@ test.onlyIfWebGL('Test vtkOpenGLTexture Rendering', (t) => {
 
   const path = `${__BASE_PATH__}/Data/skybox/mountains/`;
   const texturePathList = [];
-  texturePathList.push(`${path}px.vti`);
-  texturePathList.push(`${path}nx.vti`);
-  texturePathList.push(`${path}py.vti`);
-  texturePathList.push(`${path}ny.vti`);
-  texturePathList.push(`${path}pz.vti`);
-  texturePathList.push(`${path}nz.vti`);
+  texturePathList.push(`${path}right.jpg`);
+  texturePathList.push(`${path}left.jpg`);
+  texturePathList.push(`${path}top.jpg`);
+  texturePathList.push(`${path}bottom.jpg`);
+  texturePathList.push(`${path}front.jpg`); // front is +z on a cube
+  texturePathList.push(`${path}back.jpg`);
```

Comment:
@martinken need to remove the vti's and add the jpg's from the source?

Reply:
also, no biggie, but typo in commit title

Reply:
Yup